File: prisioner/java/src/main/java/com/everis/competition/judge/Round.java
package com.everis.competition.judge;

public class Round {
	/*
	 * This POJO class stores the basic information of a Round
	 */
	String pacoClassName;
	String antonClassName;
	int pacoPuntuation;
	int antonPuntuation;
	int id;
	public Round(String pacoClassName, String antonClassName, int scorePrisoner1, int scorePrisoner2, int id) {
		super();
		this.pacoClassName = pacoClassName;
		this.antonClassName = antonClassName;
		this.pacoPuntuation = scorePrisoner1;
		this.antonPuntuation = scorePrisoner2;
		this.id = id;
	}
	
	public String toString() { 
        String realWinner = (this.pacoPuntuation < this.antonPuntuation)? this.pacoClassName : this.antonClassName;
        String winnerString = "\n\nWinner of the round -> "+realWinner+"\n";
        if(this.pacoPuntuation == this.antonPuntuation) { winnerString = "\n\nTie in this round"; }
		return "|Round n#"+String.valueOf(this.id) +" |\n\n\tPaco was: "+this.pacoClassName+", his points were "+String.valueOf(this.pacoPuntuation) + "\n\tAnton was: "+this.antonClassName+ ", his points were "+ String.valueOf(this.antonPuntuation)+winnerString;
	}

}

File: prisioner/js/judge/Round.js
/**
 * http://usejsdoc.org/
 */

 var Round = function(pacoClassName, antonClassName, scorePrisoner1, scorePrisoner2, id) {
    this._pacoClassName = pacoClassName;
    this._antonClassName = antonClassName;
    this._pacoPuntuation = scorePrisoner1;
    this._antonPuntuation = scorePrisoner2;
    this._id = id;
 };

 Round.prototype.toString = function(){
     var realWinner = (this._pacoPuntuation < this._antonPuntuation)? this._pacoClassName : this._antonClassName;
     var winnerString = "\n\nWinner of the round -> "+realWinner+"\n";
     if(this._pacoPuntuation == this._antonPuntuation) { winnerString = "\n\nTie in this round";}
     return "|Round n#"+ this._id +" |\n\n\tPaco was: "+this._pacoClassName+", his points were "+this._pacoPuntuation + "\n\tAnton was: "+this._antonClassName+ ", his points were "+ this._antonPuntuation+winnerString;
 };
 
 module.exports.Round = Round; 